screens/LoginScreen.js

```diff
@@ -0,0 +1,131 @@
+import React, {useState} from 'react';
+import {View, StyleSheet, Platform} from 'react-native';
+import {Card, Title} from 'react-native-paper';
+import Colors from '../constants/Colors';
+import PlatformButton from '../components/PlatformButton';
+import HelperTextInput from '../components/HelperTextInput';
+import firebase from 'firebase';
+import {ChatErrors} from '../constants/Errors';
+
+const LoginScreen = ({navigation}) => {
+  const initError = {};
+  for (let key in ChatErrors.login) {
+    initError[ChatErrors.login[key]] = '';
+  }
+  const [userName, setUserName] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [currentError, setCurrentError] = useState(initError);
+
+  const loginUser = () => {
+    if (userName.trim().length >= 3) {
```

Comment:
TODO: research the use of useCallback hook. Check how many times your `PlatformButton` re-renders in current implementation and what happens if you wrap `loginUser` in `useCallback`